actions/store_command_info_MOD.js

```diff
@@ -133,32 +137,31 @@ module.exports = {
         result = jp.query(command, '$.._id');
         break;
       case 2:
-        result =
-          jp.query(command, '$..comType') === 0 || ''
-            ? 'Normal Command'
-            : jp.query(command, '$..comType') === 1
-            ? 'Includes Word'
-            : jp.query(command, '$..comType') === 2
-            ? 'Matches Regular Expression'
-            : 'Any Message';
+        {
+          const commandType = jp.query(command, '$..comType');
+          if (commandType.includes('0')) result = 'Text Command';
+          if (commandType.includes('1')) result = 'Includes Word';
+          if (commandType.includes('2')) result = 'Matches Regular Expression';
+          if (commandType.includes('3')) result = 'Any Message';
+          if (commandType.includes('4')) result = 'Slash Command';
+          if (commandType.includes('5')) result = 'User Menu Command';
+          if (commandType.includes('6')) result = 'Msg Menu Command';
```

Comment:
Can you refactor this to use a `switch` statement instead? Maybe cast the `commandType` to a number with `Number.parseInt()` and use that in the condition